exec/impl/src/main/java/org/jboss/windup/exec/WindupProcessorImpl.java

```diff
@@ -67,15 +67,16 @@ public void execute(WindupConfiguration windupConfiguration)
         {
             System.out.println("User rules dir: " + path);
             LOG.info("User rules dir: " + path);
-            LOG.warning("User rules dir: " + path);
             if (path == null)
             {
-                throw new WindupException("Null path found (all paths are: " + windupConfiguration.getAllUserRulesDirectories() + ")");
+                throw new WindupException("One of user rules dir is null."
```

Comment:
This is still not clear, and probably shouldn't throw an exception really, but that's another topic I guess. It might be better as a logged warning, either way, the message should state:

`"One or more user-supplied rule paths were null. A list of all supplied paths can be found below:\n" + windupConfiguration.getAllUserRulesDirectories()`